docs/challenges/how-challenges-work.mdx

```diff
@@ -26,10 +26,10 @@ Once you've selected a language, we'll create the git repository for you and pro
   You need to run `git` commands on your local machine.
   <Steps>
     <Step title="Open Terminal on your system">
-      If you're using Windows, make sure to [install WSL](https://learn.microsoft.com/en-us/windows/wsl/install).
+      On Windows, you can use [Command Prompt `(cmd)`](https://www.youtube.com/results?search_query=how+to+open+cmd) or [PowerShell](https://www.youtube.com/results?search_query=how+to+open+powershell), but we recommend using [WSL](https://learn.microsoft.com/en-us/windows/wsl/install).
```

Comment:
```suggestion
      If you're on Windows, you can use [Command Prompt `(cmd)`](https://www.youtube.com/results?search_query=how+to+open+cmd) or [PowerShell](https://www.youtube.com/results?search_query=how+to+open+powershell).
```

- This is still not a Windows-specific guide, so we do need to say "If you're on..."
- Let's remove WSL entirely to avoid confusion for now (once the CLI is also Windows-friendly too, we can essentially avoid recommending WSL for everything but running a program locally)

Reply:
Let's also not link to youtube. Is there no official microsoft doc we can link to?

Reply:
ms doc looks like this: https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/cmd

@rohitpaulk  My worries are for users don't know git, there is also a risk that they probably don't know how to open cmd or powershell, thus the yt search page with `how to open cmd/powershell` prefilled-in.

Reply:
Okay - can we recommend just Powershell, not cmd (let's not confuse further), and link to this then? https://learn.microsoft.com/en-us/powershell/scripting/windows-powershell/starting-windows-powershell?view=powershell-7.4#from-the-start-menu